src/components/FormContainer/FormContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import { Form, useFormikContext } from "formik";
+import * as React from "react";
+
+const FormContainer: React.FC = ({ children }) => {
+  const { status } = useFormikContext();
+
+  return (
+    <Form>
+      {status === "loading" && <p>Loading</p>}
+      {status !== "loading" && children}
```

Comment:
instead of using a string for "loading" then you should have an enum with the different status options so it is easier to maintain